gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/ManagedHttpJsonChannel.java

```diff
@@ -79,6 +88,7 @@ public synchronized void shutdown() {
       return;
     }
     try {
+      deadlineScheduledExecutorService.shutdown();
```

Comment:
Noticed that `awaitTermination()` method below has a _TODO_ and it always returns false. It's a bug we should fix but not in scope of this PR, can you please create an issue to track it?

Reply:
Created here: https://github.com/googleapis/gapic-generator-java/issues/1663